lib/client.js

```diff
@@ -99,7 +99,16 @@ Client.prototype._request = function (method, url, opts, cb) {
   });
 
   request[method](url, opts, function (err, res, body) {
-    var statsName;
+    var statsName,
+        cacheableRes = {};
+
+    if (res) {
+      cacheableRes = {
```

Comment:
Is there another name we could use for `cacheableRes`? Maybe call them `requestRes` and `flashheartRes`?